VaticanBAV/borget/BAVborget22.xml

```diff
@@ -310,14 +313,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   </objectDesc>
 
                   <handDesc>
-                    <handNote xml:id="h1" script="Ethiopic" corresp="#ms_i1 #ms_i2 #ms_i3 #ms_i4" />
-                    <handNote xml:id="h2" script="Ethiopic" corresp="#ms_i5 #ms_i6 #ms_i7 #ms_i8 #ms_i9" />
-                    <handNote xml:id="h3" script="Ethiopic" corresp="#ms_i10 #ms_i11" />
-                    <handNote xml:id="h4" script="Ethiopic" corresp="#ms_i12 #ms_i13 #ms_i14" />
+                    <handNote xml:id="h1" script="Ethiopic" corresp="#ms_i1 #ms_i2 #ms_i3 #ms_i4">
+                      <seg type="rubrication">
+                        Titles, the first one or two lines at the beginning of a section, holy names
+                      </seg>
+                    </handNote>
+                    <handNote xml:id="h2" script="Ethiopic" corresp="#ms_i5 #ms_i6 #ms_i7 #ms_i8 #ms_i9">
+                      <seg type="rubrication">
+                        Titles, the first one or two lines at the beginning of a section, holy names
+                      </seg>
+                    </handNote>
+                    <handNote xml:id="h3" script="Ethiopic" corresp="#ms_i10 #ms_i11">
```

Comment:
we have a new text, new layout, and new hand beginning with ms_i10 - sure no unit break? (probably not, Grebaut usually comments on that, but just to be sure...)

Reply:
That's a very good point, especially considering the difference in topic. Grébaut doesn't discuss it, and the quire structure is perhaps an issue. The thing would be this item spanning ff. 87v-88v, which is the end of one quire and the beginning of a next. In this case, it is "three fragments" of a magical prayer, which doesn't create so much of an issue, but without images or a clearer description it's hard to say for certain.